documentation/modules/ROOT/pages/configuration/notification.adoc

```diff
@@ -157,10 +157,18 @@ a|[source, json]
    "additional_data":{
       "connector_name":"my-connector",
       "data_collection":"table1",
-      "total_rows_scanned":"100"
+      "total_rows_scanned":"100",
+      "status":"SUCCEEDED" // <1>
    }
 }
 ----
+<1> The possible values are:
```

Comment:
I think we need an explananiton for each of the state.
@kgalieva Could you please take care of it?